app/assets/stylesheets/home.scss

```diff
@@ -20,18 +20,31 @@ body.home.index {
     }
 
     .leaflet-popup {
-      position: relative;
       background: white;
       padding: 1em;
       z-index: -100 !important;
-      margin-top: -4.75em;
+      margin-top: -(rem(104/2));
+      margin-left: 2em;
+
+      &:before {
+        content: "";
+        position: absolute;
+        left: -0.5em;
+        top: 50%;
+        margin-top: -0.55em;
+        @include triangle(1em, white, left);
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.